accelerator/config.h

```diff
@@ -28,9 +28,9 @@ extern "C" {
 
 #define TA_VERSION "tangle-accelerator/0.5.0"
 #define TA_HOST "localhost"
-#define TA_PORT "8000"
+#define TA_PORT "6969"
 #define TA_THREAD_COUNT 10
-#define IRI_HOST "localhost"
+#define IRI_HOST "node0.puyuma.org"
```

Comment:
Don't forget to recover the testing setting before pushing to github

Reply:
copy that!